Client/Frontend/Browser/Playlist/Controllers/PlaylistNewFolderView.swift

```diff
@@ -0,0 +1,198 @@
+// Copyright 2022 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import SwiftUI
+import Combine
+import Data
+import CoreData
+import Shared
+import BraveShared
+
+private struct PlaylistFolderImage: View {
+    static let cornerRadius = 10.0
+    private static let favIconSize = 16.0
+    private var thumbnailLoader: ImageLoader
+    private var favIconLoader: ImageLoader
+    
+    private var title: String?
+    @State private var thumbnail = UIImage()
+    @State private var favIcon = UIImage()
```

Comment:
Same comment as the other place the loaders are used, don't use state, used `@ObservedObject` and the Published var directly